amqtt/mqtt/protocol/handler.py

```diff
@@ -552,11 +552,6 @@ async def _send_packet(self, packet):
             )
         except (ConnectionResetError, BrokenPipeError):
             await self.handle_connection_closed()
-        except asyncio.CancelledError:
```

Comment:
I removed this block as I don't see an advantage of catching an exception and then reraising right away without providing further context.